packages/menu/src/Menu.tsx

```diff
@@ -104,14 +160,64 @@ function Menu({
     }
   };
 
-  const enabled = open || uncontrolledOpen;
+  const enabled = open;
```

Comment:
Be better to just use the `open` variable directly and not use this `enabled` variable.

Reply:
👍